html/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE HTML>
 <html>
   <head>
-    <base href="/samples/">
+    <base href="">
```

Comment:
Please revert back to `href="/samples/"` as this will be available at ....cern.ch/samples/